blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueRun.java

```diff
@@ -257,6 +265,45 @@
         public abstract Object getCause();
     }
 
+    @ExportedBean
+    public static final class BlueDownstreamRun {
```

Comment:
I would like to see this have a `_links` field. It should link to the downstream project. With this we can use existing code to generate a blueocean url.

Reply:
This was my initial intention, but `_links` is a map, so we could only have one "downstream" link. I talked it over with Mic and he agreed adding a specific property was the best way to go. I'm open to suggestions tho.

Reply:
We could extend `_links` to allow list of urls? So long as we only send an array when there's multiple values we shouldn't break existing stuff. But then, we'd have to be checking for single values as well in cases where we're expecting to see multiples. What do you think, @i386 ?

Reply:
no, in this case you are gonna have one _links field per job. So you dont n eed to have multiple links

Reply:
Yeah but there will be multiple downstream links for a single job, and the links field is a map

Reply:
I dont see that in this model?

Reply:
It's a collection: https://github.com/jenkinsci/blueocean-plugin/pull/1575/files#diff-9c0aa8614646845672acf40f503966b2R202

Reply:
I patterned it after the existing `causes` property